README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+
+> Description the project.
+
+
+## Built With
+
+- html
+- css
+
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
```

Comment:
- [ ] Kindly delete the sections that you are not using yet for a professional README file.